### PR TITLE
feat: add copy invite link

### PR DESCRIPTION
**Pull request by JasonNoonan**
A little beyond the scope of what the ticket called for, but this provides functionality for the Copy Invite Link button as well as setting up the button drawer for the Party Leader's controls area. Basically everything in here is up for debate or discussion if we have better ideas for how to fix.

### CODE REVIEW

File: lib/point_quest_web/live/quest.ex [hunk 1]
```diff
@@ -28,7 +44,7 @@ defmodule PointQuestWeb.QuestLive do
           Phoenix.PubSub.subscribe(PointQuestWeb.PubSub, quest.id)
 
           socket
-          |> assign(quest: quest, users: %{}, form: nil)
+          |> assign(quest: quest, actor: socket.assigns.current_actor, users: %{}, form: nil)
```
Comment (skbolton) [inline lib/point_quest_web/live/quest.ex]: We could always shorten current actor to just actor? Also as is @current_actor is available in render functions.

**Review by skbolton — APPROVED** [review]

File: lib/point_quest_web/live/quest.ex (hunk 1 above)
Comment (JasonNoonan) [reply lib/point_quest_web/live/quest.ex]: Oh nice I didn't realize that! I'll get it changed in the morning and re-request your review.

Comment (JasonNoonan) [reply lib/point_quest_web/live/quest.ex]: I just thought it through (apparently for the first time) and it totally makes sense lmao. I'm grabbing it off the assigns to put it back on the assigns named slightly different. I'm a fool.

Comment (JasonNoonan) [reply lib/point_quest_web/live/quest.ex]: updated

**Pull request merged**